helix-view/src/editor.rs

```diff
@@ -771,6 +775,112 @@ impl WhitespaceRender {
     }
 }
 
+#[derive(Debug, Default, Clone, PartialEq, Eq)]
+pub struct AutoSave {
+    /// Auto save after `n` milli seconds. Defaults to None.
+    pub after_delay: Option<u64>,
```

Comment:
```suggestion
    pub after_delay: AutoSaveAfterDelay,
```

To ensure `:set auto_save.after_delay.enable true/false` (same for the delay) works  this struct should be the same as the toml definition.

Then you can also change the enum variant below to:

``` rust
    AusoSave(AutoSave)
```